test/sample-applications/mysql-persistent-csi.yaml

```diff
@@ -0,0 +1,182 @@
+apiVersion: v1
+kind: List
+items:
+  - kind: Namespace
+    apiVersion: v1
+    metadata:
+      name: mysql-persistent
+      labels:
+        app: mysql
+  - apiVersion: v1
+    kind: ServiceAccount
+    metadata:
+      name: mysql-persistent-sa
+      namespace: mysql-persistent
+      labels:
+        component: mysql-persistent
+  - kind: SecurityContextConstraints
+    apiVersion: security.openshift.io/v1
+    metadata:
+      name: mysql-persistent-scc
+    allowPrivilegeEscalation: false
+    allowPrivilegedContainer: false
+    volumes:
+    - '*'
+    users:
+    - system: nacuser
```

Comment:
if created user has different name, it will work? (should this have space between : ?)